dashboards/lodestar_validator_monitor.json

```
@@ -167,7 +167,7 @@
             "uid": "${DS_PROMETHEUS}"
           },
           "exemplar": false,
-          "expr": "validator_monitor_validators_status",
+          "expr": "validator_monitor_validators_statuses",
```

not native english speaker but double plural sounds odd to me

```suggestion
          "expr": "validator_monitor_validator_statuses",
```